src/App.js

```diff
@@ -1,25 +1,63 @@
-import logo from './logo.svg';
+import React, { Component } from 'react'
 import './App.css';
+import PokeList from './PokeList.js'
+import Dropdown from './Dropdown.js';
+class App extends Component {
+  state = { 
+    data: [],
+    loading: true,
+    query: null,
+    option: ''
+  }
+  sortOption = ['asc', 'desc']
+  ascDescOrderChange = (e) => {
+    this.setState({option: e.target.value})
+  }
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  fetchPokemon = async () => {
+    let url = 'https://pokedex-alchemy.herokuapp.com/api/pokedex';
+    if (this.state.query) {
+      url = url + `?pokemon=${this.state.query}`;
+    }
+    if (this.state.option) {
+      url = url + `?sort=pokemon&direction=${this.state.option}`
+    }
+    let response = await fetch(url);
+
+    let data = await response.json();
+    this.setState({ data: data.results, loading: false});
+  };
+
+  updateQuery = (e) => {
+    this.setState({ query: e.target.value });
+  };
 
+  componentDidMount() {
+    this.fetchPokemon();
+  }
+
+  render() { 
+    const { loading } = this.state;
+    return (
+      <>
+        <h1>Pokemon!</h1> 
+        {loading && <h3> LOADING </h3>}
```

Comment:
You want to use an actual loading spinner animation rather than simply showing the word 'Loading'.